modules/machine-learning-services/workspace/README.md

```diff
@@ -19,7 +19,7 @@ This module deploys a Machine Learning Services Workspace.
 | `Microsoft.Insights/diagnosticSettings` | [2021-05-01-preview](https://learn.microsoft.com/en-us/azure/templates/Microsoft.Insights/2021-05-01-preview/diagnosticSettings) |
 | `Microsoft.MachineLearningServices/workspaces` | [2022-10-01](https://learn.microsoft.com/en-us/azure/templates/Microsoft.MachineLearningServices/2022-10-01/workspaces) |
 | `Microsoft.MachineLearningServices/workspaces/computes` | [2022-10-01](https://learn.microsoft.com/en-us/azure/templates/Microsoft.MachineLearningServices/2022-10-01/workspaces/computes) |
-| `Microsoft.Network/privateEndpoints` | [2022-07-01](https://learn.microsoft.com/en-us/azure/templates/Microsoft.Network/2022-07-01/privateEndpoints) |
+| `Microsoft.Network/privateEndpoints` | [2022-09-01](https://learn.microsoft.com/en-us/azure/templates/Microsoft.Network/2022-09-01/privateEndpoints) |
```

Comment:
@Sdelausnay looks good and thanks! I'd suggest to prioritize #3665 before this one, which also takes care to update all outdated api versions

Reply:
yes sure no problem, will merge the changes back into this one once it is completed.

Reply:
@Sdelausnay we hat the merge the PR #3665 into an intermediate branch first and open a new PR #3778 to combine some fixes and successfully test the pipelines. I blieve there is just one issue remaining and then we can merge to main.

Reply:
Aaand done